config/locales/authentication.yml

```diff
@@ -9,13 +9,11 @@ en:
       email:
         subject: Please confirm your email address
         heading: Please confirm your email address
-        body: Click the link below to sign in to the Apply for postgraduate teacher training service.
     check_your_email: Check your email
     sign_in:
       heading: Sign In
       email:
-        subject: Sign In
-        heading: Please confirm you want to Sign In
-        body: Click the link below to sign in to the Apply for postgraduate teacher training service.
+        subject: Sign in to Apply for teacher training
+        heading: Sign in to Apply for teacher training
```

Comment:
are we using heading?

Reply:
I've updated this to remove `heading` and use the `subject` translation string in the email for the heading of it.